### ISSUE TITLE
using C# property/field in Javascript .net 

### ISSUE THREAD

**Issue by Hajer10** [paulbartrum/jurassic#94]
hi, I was trying to find a way to link my property in a function of javascript, is it possible to di that?

**Comment by paulbartrum** [comment]
Yes, there is a `[JSProperty]` attribute that you can use in the same way as [JSMethod].
